README.md

```diff
@@ -18,10 +22,10 @@ In order to set up: make sure that Node Health Check is running, Machine-deletio
 ## Example CRs
 An example MDR template object.
 ```yaml
-   apiVersion: machine.remediation.medik8s.io/v1alpha1
-   kind: MachineDeletionRemediationTemplate
+   apiVersion: machine-deletion.medik8s.io/v1alpha1
```

Comment:
this should be machine-deletion-remediation, I think. It was already changed in a previous PR.
The same goes for similar changes below.